src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6340,6 +6340,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_NonPrivateAPIInRecord" xml:space="preserve">
     <value>Record member '{0}' must be private.</value>
   </data>
+  <data name="WRN_PrecedenceInversion" xml:space="preserve">
+    <value>Operator cannot be used here due to precedence. Use parentheses to disambiguate.</value>
```

Comment:
Consider including the operator token in the diagnostic message, e.g. "Operator 'as' cannot be used here due to precedence.."